cachito/workers/pkg_managers/general.py

```diff
@@ -179,18 +179,18 @@ async def async_download_binary_file(
     :raise NetworkError: If download failed
     """
     try:
-        log.debug(f"Download started - {tarball_name}")
         async with session.get(url, auth=auth, raise_for_status=True) as resp:
             with open(os.path.join(download_dir, tarball_name), "wb") as f:
                 while True:
                     chunk = await resp.content.read(chunk_size)
                     if not chunk:
                         break
                     f.write(chunk)
-    except Exception as e:
-        raise NetworkError(f"Could not download {url}: {e}")
 
-    log.debug(f"Download completed - {tarball_name}")
```

Comment:
Maybe we could keep the logs about starting and completing the downloads? I think it's good to have them as a progress indicator

Reply:
Yeah, but it seemed like a very big spam in big requests. Don't you think that would be a problem?

Reply:
IMO this level of spam is acceptable for debug logs. If I'm not mistaken, we did log all the downloads before converting to asyncio no? We also log all downloads for pip I believe.

Maybe keep just only the completed message?

Reply:
If this is a behavior that already happens in pip, I don't see a problem in adding it.